compiler/p/codegen/OMRRealRegister.hpp

```diff
@@ -78,12 +78,13 @@ class OMR_EXTENSIBLE RealRegister : public OMR::RealRegister
       pos_CX     = 3
    } PPCOperandPosition;
 
-   typedef enum {
-      FPCC_FL = 0x0, // less than
-      FPCC_FG = 0x1, // greater than
-      FPCC_FE = 0x2, // equal
-      FPCC_FU = 0x3  // unordered
-   } PPCFloatingPointConditionCode;
+   enum CRCC {
+      //Condition Register Condition Code (CRCC)
+      CRCC_LT = 0x0, // less than
+      CRCC_GT = 0x1, // greater than
+      CRCC_EQ = 0x2, // equal
+      CRCC_SO = 0x3  // summary overflow
```

Comment:
What do you think about adding an alias for `SO` to represent `FU` and using that when dealing with FP regs? Something like:

```
enum CRCC {
       //Condition Register Condition Code (CRCC)
       CRCC_LT = 0x0,    // less than
       CRCC_GT = 0x1,    // greater than
       CRCC_EQ = 0x2,    // equal
       CRCC_SO = 0x3,    // summary overflow
       CRCC_FU = CRCC_SO // floating-point unordered
};
```

That would make the intention of the code clearer to anyone reading it if they didn't already know that the `SO` and `FU` bits were the same.